set-theory.html.markdown

```diff
@@ -1,143 +1,194 @@
-* Items in a set are not order sensitive. `{1,2,3,4}={2,3,1,4}`
-* Items in a set are unique. `{1,2,2,3,4,3,4,2}={1,2,3,4}`
-* Two sets are equal if and only if all of their items are exactly equal to each other. Representation: `A=B`. Otherwise, they are not equal. Representation: `A≠B`.
-* `A=B` if `A ⊆ B` and `B ⊆ A`
-
-### Belongs to
-* If the set A contains an item `x`, `x` belongs to A (`x∈A`).
-* Otherwise, `x` does not belong to A (`x∉A`).
-
-### Subsets
-* If all items in a set `B` are items of set `A`, we say that `B` is a subset of `A` (`B⊆A`).
-* If B is not a subset of A, the representation is `B⊈A`.
-
-### Proper subsets
-* If `B ⊆ A` and `B ≠ A`, B is a proper subset of A (`B ⊂ A`). Otherwise, B is not a proper subset of A (`B ⊄ A`).
-
-## Set operations
-### Base number
-* The number of items in a set is called the base number of that set. Representation: `|A|`
-* If the base number of the set is finite, this set is a finite set.
-* If the base number of the set is infinite, this set is an infinite set.
-```
-A   = {A,B,C}
-|A| = 3
-B   = {a,{b,c}}
-|B| = 2
-|∅| = 0         (it has no items)
-```
-
-### Powerset
-* Let `A` be any set. The set that contains all possible subsets of `A` is called a powerset (written as `P(A)`).
-```
-P(A) = {x|x ⊆ A}
-
-|A| = N, |P(A)| = 2^N
-```
-
-## Set operations among two sets
-### Union
-Given two sets `A` and `B`, the union of the two sets are the items that appear in either `A` or `B`, written as `A ∪ B`.
-```
-A ∪ B = {x|x∈A∨x∈B}
-```
-### Intersection
-Given two sets `A` and `B`, the intersection of the two sets are the items that appear in both `A` and `B`, written as `A ∩ B`.
-```
-A ∩ B = {x|x∈A,x∈B}
-```
-### Difference
-Given two sets `A` and `B`, the set difference of `A` with `B` is every item in `A` that does not belong to `B`.
-```
-A \ B = {x|x∈A,x∉B}
-```
-### Symmetrical difference
-Given two sets `A` and `B`, the symmetrical difference is all items among `A` and `B` that doesn't appear in their intersections.
-```
-A △ B = {x|(x∈A∧x∉B)∨(x∈B∧x∉A)}
-
-A △ B = (A \ B) ∪ (B \ A)
-```
-### Cartesian product
-Given two sets `A` and `B`, the cartesian product between `A` and `B` consists of a set containing all combinations of items of `A` and `B`.
-```
-A × B = { {x, y} | x ∈ A, y ∈ B }
-```
-## "Generalized" operations
-### General union
-Better known as "flattening" of a set of sets.
-```
-∪A = {x|X∈A,x∈X}
-∪A={a,b,c,d,e,f}
-∪B={a}
-∪C=a∪{c,d}
-```
-### General intersection
-```
-∩ A = A1 ∩ A2 ∩ … ∩ An
-```
+
+## Introduction
+
+Set theory is the mathematical study of collections of objects. These
+objects can be anything from numbers, computer programs, beach balls,
+or other sets. Axiomatic set theory forms the foundation of mathematics,
+and set theory appears everywhere in Computer Science.
+
+Set theory as we know it was innovated in the 19th century by
+[Georg Cantor](https://en.wikipedia.org/wiki/Georg_Cantor),
```

Comment:
I don't think that this historic information is necessary for the LXIYM format; people can just go straight to Wikipedia